### PR TITLE
Public portal dev

### PR DESCRIPTION
**Pull request by anubhavitis**

### CODE REVIEW

File: src/domain/publicPortal/create.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+}
+
+async function update(fileId, contractAddress, files) {
+    const portal = await Portal.findOneAndUpdate({ fileId, contractAddress }, { files });
+    return portal;
+}
+
+async function getPortalOps(fileId, contractAddress) {
+    const portal = await findOne(fileId, contractAddress);
+    return portal ? update : create;
```
Comment (nadeem-fileverse) [inline src/domain/publicPortal/create.js:16]: I don't think you need two separate functions for this `findOneAndUpdate` has an option to create new if it does not find data in db
https://mongoosejs.com/docs/tutorials/findoneandupdate.html

File: src/infra/database/models/portal.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
+const mongoose = require('mongoose');
+const { Schema } = mongoose;
+
+const _portal = {};
+
+_portal.schema = new Schema({
```
Comment (nadeem-fileverse) [inline src/infra/database/models/portal.js:6]: can you share a example data set that has been saved in the DB for this schema want to take a look at the data

**Review by nadeem-fileverse — CHANGES_REQUESTED** [review]
PTAL

File: src/domain/publicPortal/create.js (hunk 1 above)
Comment (anubhavitis) [reply src/domain/publicPortal/create.js:16]: oh wow. Didn't know that. 
Removing the other func.

**Review by nadeem-fileverse — APPROVED** [review]
LGTM

**Pull request merged**